### PR TITLE
Adding the first commit with the full project upload

### PR DESCRIPTION
**Pull request by amitkumardianapps**
Added all the project files and requesting review for the same

### CODE REVIEW

File: .idea/encodings.xml
Comment (ManishDua90) [inline .idea/encodings.xml:1]: .idea folder should not be committed

add in .gitignore

File: src/main/java/helpers/ExtentManager.java
Comment (ManishDua90) [inline src/main/java/helpers/ExtentManager.java:1]: add under reporting

File: src/main/java/helpers/ExtentTestManager.java
Comment (ManishDua90) [inline src/main/java/helpers/ExtentTestManager.java:1]: add under reporting

File: src/main/java/helpers/TestListener.java
Comment (ManishDua90) [inline src/main/java/helpers/TestListener.java:1]: add under reporting 
rename

File: src/main/java/org/activity/task/Main.java
Comment (ManishDua90) [inline src/main/java/org/activity/task/Main.java:1]: delete

File: src/main/java/page/Cart.java
Comment (ManishDua90) [inline src/main/java/page/Cart.java:1]: page objects

File: src/test/java/resources/testng.xml
Comment (ManishDua90) [inline src/test/java/resources/testng.xml:1]: tags
groups

regression,sanity,.....

File: src/test/java/tests/CartTest.java
Comment (ManishDua90) [inline src/test/java/tests/CartTest.java:1]: rename

File: pom.xml
Comment (ManishDua90) [inline pom.xml:1]: group id incorrect
artifactid incorrect

remove comments

File: .idea/encodings.xml
Comment (ManishDua90) [reply .idea/encodings.xml:1]: feature/.....
bugfix/......
hotfix/........

Comment (ManishDua90) [reply .idea/encodings.xml:1]: git checkout feature/exercise

File: src/test/java/tests/CartTest.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+public class CartTest {
+
+  @BeforeClass
+  public static void login() {
+    BrowserSetup.setup();
+  }
+
+  @Test(dataProvider = "LoginCredentials", dataProviderClass = LoginData.class)
+  public static void loginPage(String username, String password) {
```
Comment (ManishDua90) [inline src/test/java/tests/CartTest.java]: method name incorrect

File: src/test/java/tests/CartTest.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    LoginOperations.login(username, password);
+    ProductsOperations.addToCart();
```
Comment (ManishDua90) [inline src/test/java/tests/CartTest.java]: Hardcoded products

Comment (ManishDua90) [inline src/test/java/tests/CartTest.java]: rename method

File: src/test/java/tests/CartTest.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    CartOperations.clickCart();
+    CartOperations.verifyHeading();
```
Comment (ManishDua90) [inline src/test/java/tests/CartTest.java]: name 
assertion methods should be clubbed with operations - Logaical

File: src/main/java/helpers/BrowserSetup.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class BrowserSetup {
+
+  public static WebDriver driver = new ChromeDriver();
+
+  public static void setup() {
+    WebDriverManager.chromedriver().setup();
+    driver.get("https://www.saucedemo.com/v1/");
```
Comment (ManishDua90) [inline src/main/java/helpers/BrowserSetup.java:16]: hardcoding

Comment (ManishDua90) [reply src/main/java/helpers/BrowserSetup.java:16]: create constants package

Comment (ManishDua90) [reply src/main/java/helpers/BrowserSetup.java:16]: Add logging in all methods

File: src/main/java/helpers/ExtentManager.java
Comment (amitkumardianapps) [reply src/main/java/helpers/ExtentManager.java:1]: Done

**Pull request merged**